script.js

```diff
@@ -46,4 +46,85 @@ closeButton.addEventListener('click', closePop);
+    live: 'https://whoistolu.github.io/personal-portfolio/',
+    source: 'https://github.com/Whoistolu/personal-portfolio',
+  },
+  {
+    name: 'Multi School Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry',
+    image: 'img/ten.png',
+    technologies: ['ruby', 'css', 'javascript', 'bootsrap'],
+    live: 'https://whoistolu.github.io/personal-portfolio/',
+    source: 'https://github.com/Whoistolu/personal-portfolio',
+  },
+  {
+    name: 'Multi School Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry',
+    image: 'img/ten.png',
+    technologies: ['ruby', 'css', 'javascript', 'bootsrap'],
+    live: 'https://whoistolu.github.io/personal-portfolio/',
+    source: 'https://github.com/Whoistolu/personal-portfolio',
+  },
+  {
+    name: 'Multi School Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry',
+    image: 'img/ten.png',
+    technologies: ['ruby', 'css', 'javascript', 'bootsrap'],
+    live: 'https://whoistolu.github.io/personal-portfolio/',
+    source: 'https://github.com/Whoistolu/personal-portfolio',
+  },
+  {
+    name: 'Multi School Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry',
+    image: 'img/ten.png',
+    technologies: ['ruby', 'css', 'javascript', 'bootsrap'],
+    live: 'https://whoistolu.github.io/personal-portfolio/',
+    source: 'https://github.com/Whoistolu/personal-portfolio',
+  },
+  {
+    name: 'Multi School Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry and typesetting industry. Lorem Ipsum has been the industry',
+    image: 'img/ten.png',
+    technologies: ['ruby', 'css', 'javascript', 'bootsrap'],
+    live: 'https://whoistolu.github.io/personal-portfolio/',
+    source: 'https://github.com/Whoistolu/personal-portfolio',
+  },
+];
```

Comment:
- [ ] Kindly refactor this code, you can use a counter to create copies of the same project or better yet create a different project  to vary the projects array

Reply:
## Hello, [@AshakaE](https://github.com/AshakaE)

The project is not supposed to have the same details hence, I have changed the project details in the array.
Thank you for the feedback.